recipes/abseil/all/conandata.yml

```
@@ -15,6 +18,12 @@ sources:
     url: "https://github.com/abseil/abseil-cpp/archive/20200225.3.tar.gz"
     sha256: "66d4d009050f39c104b03f79bdca9d930c4964016f74bf24867a43fbdbd00d23"
 patches:
+  "20220623.1":
+    - patch_file: "patches/0003-absl-string-libm.patch"
+    - patch_file: "patches/0005-has-unique-object-representations.patch"
+      patch_description: "Workaround bug in GCC 7.2"
+      patch_source: "https://github.com/abseil/abseil-cpp/pull/1250"
+      patch_type: "backport"
```

Backport is no longer accepted  since #14096 -- the linter doesn't leave enough comments 😑 
```suggestion
      patch_type: "bugfix"
```

---

seems to be portability instead, not a bugfix, as per description of patch types.

---

Good catch space!